controllers/generic_service_handler.go

```diff
@@ -185,7 +186,7 @@ func (h *genericServiceHandler) get(ctx context.Context, avn *aiven.Client, avnG
 		return nil, nil
 	}
 
-	status.State = "RUNNING" // overrides REBALANCING
+	status.State = service.ServiceStateType("RUNNING") // overrides REBALANCING
```

Comment:
I think, no we can just use this
https://github.com/aiven/go-client-codegen/blob/main/handler/service/service.go#L1596